src/index.jsx

```diff
@@ -25,30 +25,26 @@ subscribe(APP_READY, () => {
       <HelmetProvider>
         <Head />
         <Header />
-        {getConfig().USE_LR_MFE ? (
-          <Routes>
+        <Routes>
+          <Route
+            path={ROUTES.PROGRAM_RECORDS}
+            element={<AuthenticatedPageRoute><ProgramRecordsList /></AuthenticatedPageRoute>}
+          />
+          <Route
+            path={ROUTES.PROGRAM_RECORD_SHARED}
+            element={<ProgramRecord isPublic />}
+          />
+          <Route
+            path={ROUTES.PROGRAM_RECORD_ITEM}
+            element={<AuthenticatedPageRoute><ProgramRecord isPublic={false} /></AuthenticatedPageRoute>}
+          />
+          {getConfig().ENABLE_VERIFIABLE_CREDENTIALS && (
```

Comment:
I assume it's fine that you moved this? The motivation for it isn't clear.

Reply:
It was originally conditionally rendered _inside_ of the the other conditional (the route to `verifiable credentials` would only be enabled iff both flags were enabled). I just moved it up a layer (and at the bottom of all the other routes)